.github/workflows/checks.yaml

```diff
@@ -56,6 +56,96 @@ jobs:
             authz
             examples
 
+  associated-issue:
+    name: Issue Accociated with Pull Request
+    if: contains(fromJSON('["pull_request", "pull_request_target"]'), github.event_name)
```

Comment:
```suggestion
    if: >-
      contains(fromJSON('["pull_request", "pull_request_target"]'), github.event_name)
      && !contains(fromJSON('["dependabot", "opentdf-automation"]'), github.actor)
```

Reply:
I updated the if condition with what I think will work. Not an easy way to test this until we merge in I think.

Reply:
I updated the if condition with what I think will work. Not an easy way to test this until we merge in I think.